packages/model-eval/src/radiantModels.ts

```diff
@@ -0,0 +1,26 @@
+/**
+  @fileoverview This file contains the list of LLMs that are available through Radiant.
+ */
+// TODO: get more models
+export const radiantModels: {
+  label: string;
+  radiantModelDeployment: string;
+}[] = [
+  {
+    label: "gpt-35-turbo",
+    radiantModelDeployment: "gpt-35-turbo-eai-experimentation",
+  },
+  {
+    label: "gpt-4",
+    radiantModelDeployment: "gpt-4-eai-experimentation",
+  },
+  {
+    label: "gpt-4o",
+    radiantModelDeployment: "gpt-4o-eai-experimentation",
+  },
+  // TODO: mistral seems broken
+  // {
+  //   label: "mistral-large",
+  //   radiantModelDeployment: "Mistral-large-eai",
+  // },
```

Comment:
EAI for this?

Reply:
there's no specific ticket on our end for this issue, but i think it's generally captured by https://jira.mongodb.org/browse/EAI-379, which is to use more models on the evals